pkgs/tools/text/cringify/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+}:
+
+rustPlatform.buildRustPackage {
+  pname = "cringify";
+  version = "1.1";
```

Comment:
It's a bit confusing with the version of this package. In the `cargo.toml` and `cargo.lock` files it's version is listed as 0.1.1 (not 1.1), while when calling `cringify -V` it tells me it's version is 0.1.0.
Either way, 1.1 seems to not be the correct version.

Reply:
-V had to be patched, as upstream has not updated it